csuntechlab/iSTART
# STR-139 -- DO:Remove Laravel Echo & Create a Local DB through Docker

# Description
- Removed all Laravel Echo instances/dependencies.
- Added Redis server served inside a docker container on port 6379
# Testing
- Run docker-compose up -d 
- Wait a few minutes
- Now run `docker logs istart_redis' to see if redis is served up
<img width="607" alt="Screen Shot 2019-05-01 at 2 36 11 PM" src="https://user-images.githubusercontent.com/36258268/57044176-83f1ee80-6c1e-11e9-9304-65718e42d3f7.png">
- Your redis docker container should now be up 👍
